plasmapy/physics/parameters.py

```diff
@@ -1424,6 +1456,12 @@ def lower_hybrid_frequency(B, n_i, ion='p+'):
         which defaults to protons.  If no charge state information is
         provided, then the ions are assumed to be singly charged.
 
+    to_hz : bool, optional
+        By default the output of this function will return the gyrofrequency in
+        radians per second. However, it can also be expressed in units of Hertz.
+        Default is 'False', if it is changed to True decorator angular_freq_to_hz
+        will convert it to Hertz.
+
```

Comment:
If the decorator with the `+=` parameters works as intended, we can get rid of this added docstring note wherever it's used - otherwise we're gonna see `to_hz` twice.